front/cypress/e2e/admin/pages_and_menu/update_homepage_hero_banner_content.cy.ts

```diff
@@ -102,6 +102,7 @@ describe('Admin: update Hero Banner content', () => {
       'testimage.png'
     );
     cy.get('[data-cy="e2e-image-cropper"]'); // wait until image cropper is loaded and visible (image is cropped). The test is flaky without this line.
+    cy.get('#overlay-toggle').should('not.exist');
```

Comment:
Added the overlay toggle checks in the E2E because it was very hard to test with React Testing Library.

Reply:
Good question. Changing this.